subprojects/core-api/src/main/java/org/gradle/api/artifacts/repositories/MavenArtifactRepository.java

```diff
@@ -127,5 +127,26 @@
         void artifact();
     }
 
+    /**
+     * The different common types of Maven repositories.
+     *
+     * @since 5.1
+     */
+    @Incubating
```

Comment:
Can we simplify these enums to be one word only?
```
    @Incubating
    enum RepositoryKind {
        RELEASES
        SNAPSHOTS,
        BOTH
    }
```

Reply:
Surely we can. If it's only about cosmetics ;) The reason to use `MIXED` instead of `BOTH` is that `BOTH` implies that you know what the 2 are. Which may be obvious for you, but not for us (may be more values later).